tests/publiccloud/create_aistack_env.pm

```diff
@@ -28,38 +28,43 @@ use File::Basename;
 use version_utils;
 use Data::Dumper;
 
+sub test_flags {
+    return {fatal => 1, publiccloud_multi_module => 1};
+}
+
 sub install_dependency_package {
     my ($instance) = @_;
     my $rke2_url = get_var('RKE2_URL');
     my $kubectl_url = get_var('KUBECTL_URL');
     my $helm_url = get_var('HELM_URL');
 
     record_info('Dep pkg install');
-    trup_call("pkg install curl git docker");
+    trup_call("pkg install curl git podman python311");
 
     # docker activation section
     process_reboot(trigger => 1);
-    systemctl("enable docker");
-    systemctl("start docker");
-    systemctl("status docker");
+    systemctl("enable podman");
+    systemctl("start podman");
+    systemctl("status podman");
```

Comment:
Are you sure you need this? I don't know how podman integrates with RKE2 but podman generally works serverless.

Reply:
for my future feature test..I need this.